docs/instruction/03-runtime.md

```diff
@@ -1,215 +1,270 @@
 
-View the output log:
+### 2.4 Package Plugins
 
-```console
-tail -f logs/eventmesh.out
+The `installPlugin` task will copy the built plugins to the `dist` directory:
+
+```shell
+./gradlew installPlugin
 ```
 
-![runtime_3](/images/install/runtime_3.png)
+EventMesh will load the plugins from the `plugin` directory based on the configuration in `eventmesh.properties`.
 
-## 2 Remote deployment
+After a successful build, the `dist` directory in the project root contains the binary files for EventMesh. For configuration and startup, refer to [Binary Distribution Deployment](#1-binary-distribution-deployment).
 
-### 2.1 Dependencies
+## 3. Start from Source Code
 
-- 64-bit OS, we recommend Linux/Unix.
+### 3.1 Dependencies
 
-- 64-bit JDK 1.8 or JDK 11
+- Recommended to use 64-bit Linux/Unix systems
+- 64-bit JDK 8 or JDK 11
+- [Gradle](https://docs.gradle.org/current/userguide/installation.html) 7.0+ (optional), the build commands provided in this document use the Gradle Wrapper, and you don't need to configure the Gradle environment yourself. You can also check the recommended Gradle version for your EventMesh version in the `gradle/wrapper/gradle-wrapper.properties` file and use your local Gradle version for compilation.
+- It is recommended to use an IDE (Integrated Development Environment) to import EventMesh. `Intellij IDEA` is recommended as the IDE.
 
-- Gradle 7.0+, The current recommended version can be found in the `gradle/wrapper/gradle-wrapper.properties` file.
+### 3.2 Download
 
-### 2.2 Download
+Clone the code from GitHub:
+
+```shell
+git clone https://github.com/apache/eventmesh.git
+cd eventmesh/
+```
 
-Download and extract the executable binaries of the latest release from [EventMesh download](https://eventmesh.apache.org/download).For example, with the current latest version, you will get `apache-eventmesh-1.9.0.tar.gz`.
+You can also download the Source Code release from [EventMesh Download](https://eventmesh.apache.org/download) and extract it:
 
-```console
-tar -xvzf apache-eventmesh-1.9.0-bin.tar.gz
-cd apache-eventmesh-1.9.0
+```shell
+wget https://dlcdn.apache.org/eventmesh/1.10.0/apache-eventmesh-1.10.0-source.tar.gz
+tar -xvzf apache-eventmesh-1.10.0-source.tar.gz
+cd apache-eventmesh-1.10.0-src/
 ```
 
-### 2.3 Deploy
+### 3.3 Project Structure Explanation
 
-Edit the `eventmesh.properties` to change the configuration (e.g. TCP port, client blacklist) of EventMesh Runtime. The executable binaries contain all plugins in the bundle, thus there's no need to build them from source code.
+| Module Name                | Description                          |
+| -------------------------- | ------------------------------------ |
+| eventmesh-common           | EventMesh common classes and methods module |
+| eventmesh-connector-api    | EventMesh Connector plugin interface definition module |
+| eventmesh-connector-plugin | EventMesh Connector plugin module    |
```

Comment:
The `eventmesh-connector-api` and `eventmesh-connector-plugin` modules are no longer present in the project.

Reply:
Updated in https://github.com/apache/eventmesh-site/pull/170/commits/dad1f5807f4146a7b06f8eb45e8cb350f5e7f6ae.